Zend/tests/weakrefs/weakrefs_008.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+WeakReference no references
+--FILE--
+<?php
+$wr = new WeakReference(new stdClass);
+
+$disallow = &$wr->disallowed;
+?>
+--EXPECTF--
+Fatal error: Uncaught Error: WeakReference objects do not support property references in %s:4
+Stack trace:
+#0 {main}
+  thrown in %s on line 4
```

Comment:
Please combine test 002 and 003, as well as 004-008 into a single file each (with Errors being caught).

Windows apparently has a huge per-test cost, so we should take a bit of care to not split up tests to an unreasonable degree.